.travis.yml

```diff
@@ -22,7 +22,7 @@ services:
 
 env:
   global:
-  - CORE_BRANCH=master
+  - CORE_BRANCH=stable12
```

Comment:
But then it is only tested against stable12 and not against master anymore?! Why not add an additional test run?